src/core/src/OmniPolygonRasterOverlay.cpp

```diff
@@ -104,6 +116,14 @@ void OmniPolygonRasterOverlay::reload() {
 
     _pPolygonRasterOverlay = new CesiumRasterOverlays::RasterizedPolygonsOverlay(
         rasterOverlayName, polygons, getInvertSelection(), *pEllipsoid, projection, options);
+
+    auto cesiumPolygonRasterOverlay = UsdUtil::getCesiumPolygonRasterOverlay(_pContext->getUsdStage(), _path);
+    bool excludeSelectedTiles;
+    cesiumPolygonRasterOverlay.GetExcludeSelectedTilesAttr().Get(&excludeSelectedTiles);
+    if (excludeSelectedTiles) {
+        auto pPolygons = _pPolygonRasterOverlay.get();
+        _pExcluder = std::make_shared<Cesium3DTilesSelection::RasterizedPolygonsTileExcluder>(pPolygons);
+    }
```

Comment:
Could be condensed to something like:

```c++
    if (getExcludeSelectedTiles()) {
        _pExcluder = std::make_shared<Cesium3DTilesSelection::RasterizedPolygonsTileExcluder>(_pPolygonRasterOverlay.get());
    }
```

Reply:
Yes, the accessor is better. I added that before the accessor. It's probably good practice to set up every accessor for USD properties of prims when creating the Omni*.cpp classes.